test/sqatt/src/Sqatt.hs

```diff
@@ -531,3 +540,15 @@ mkLogDir strPrefix = do
         currDateStr = toFSSafeStr (show currDate)
     mktree logDir
     return logDir
+
+emptyExample :: TxsExample
+emptyExample = TxsExample
```

Comment:
Why is emptyExample needed?
It doesn't seem to be related to parameters, but to setup and teardown actions.

Reply:
Ah, that is something I forgot... I added setup and teardown actions because I have figured they're needed for testing this change. But I didn't add unit tests yet; that was the reason I have estimated 3 days. OK, more to be done here...